### PR TITLE
fix: rangefilter will not force values on user

### PR DESCRIPTION
**Pull request by MaxPostema**
https://docs.google.com/document/d/1GYEliXqfVSVTOYV5kYlM4Vnm80j3vbvPr6azdilB3pU/edit#

Min value input field can hold numbers bigger than max value input field. Forcing the correct values let to poor user experience. 

#### Checklist
- [ ] Functionality works & meets specifications
- [x] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] Conventional commits (squash if needed)
- [ ] No warnings during install
- [ ] Updated javascript typing

### CODE REVIEW

File: src/components/ActiveFilters.vue [hunk 1]
```diff
@@ -92,11 +92,25 @@ export default Vue.extend({
...
           if (filter.type === 'range-filter') {
-            activeValues.push({
-              key,
-              value: `${current[0]} to ${current[1]}`,
-              label: filter.label
-            })
+            if (current[0] == null && current[1] != null) {
+              activeValues.push({
+                key,
+                value: `${current[1]} and less`,
```
Comment (connoratrug) [inline src/components/ActiveFilters.vue:98]: 'and less' i18n ? , or do we do this later in separate PR for all text ?

File: src/components/filters/RangeFilter.vue [hunk 2]
```diff
@@ -1,27 +1,47 @@
 
 <template>
   <div>
-    <b-input-group>
+    <b-input-group class="mb-1">
```
Comment (connoratrug) [inline src/components/filters/RangeFilter.vue:4]: only for range filter ?

File: src/components/filters/RangeFilter.vue [hunk 3]
```diff
@@ -76,19 +100,29 @@ export default Vue.extend({
...
-      this.sliderValue[1] = Math.max(newValue[0], newValue[1])
+      if (newValue[0] == null && newValue[1] == null) {
+        this.sliderValue = [null, null]
+      }
     }
   },
   methods: {
+    setSliderValue (id, value) {
+      this.sliderValue[id] = value
+      this.handleChange()
```
Comment (connoratrug) [inline src/components/filters/RangeFilter.vue:111]: handleSliderValueChange ? or just add to setSliderValue ?

File: src/components/filters/RangeFilter.vue [hunk 4]
```diff
@@ -76,19 +100,29 @@ export default Vue.extend({
...
-      if (this.sliderValue[0] === this.min && this.sliderValue[1] === this.max) {
-        this.$emit('input', undefined)
-      } else {
-        this.sliderValue = [parseFloat(Math.max(this.min, this.sliderValue[0])), parseFloat(Math.min(this.max, this.sliderValue[1]))]
-        // clone to break reactive loop
-        this.$emit('input', [...this.sliderValue])
+      let minVal = null
+      let maxVal = null
+      if (this.sliderValue[0] != null) {
+        minVal = parseFloat(Math.min(Math.max(this.sliderValue[0], this.min), this.max))
```
Comment (connoratrug) [inline src/components/filters/RangeFilter.vue:117]: 2 time .max ? is that what we want, maybe add a comment on why this is , not clear to me by looking at the code, but that could just be me ;)

**Comment by connoratrug** [comment]
As user i can't see the max ( or min ) range , when i enter a value above the max , the value jumps to max this seems confusing to me ( maybe have look at lifelines age filter ?)

File: src/components/filters/RangeFilter.vue (hunk 2 above)
Comment (MaxPostema) [reply src/components/filters/RangeFilter.vue:4]: Only for the first one. A but of space below it makes it look better.

File: src/components/filters/RangeFilter.vue (hunk 3 above)
Comment (MaxPostema) [reply src/components/filters/RangeFilter.vue:111]: Improved naming

File: src/components/ActiveFilters.vue (hunk 1 above)
Comment (MaxPostema) [reply src/components/ActiveFilters.vue:98]: There seems to be no i18n in the entire project.

Comment (MaxPostema) [reply src/components/ActiveFilters.vue:98]: Added the point to:
https://wiki.gcc.rug.nl/ticket/7326

**Pull request merged**